pkg/gadgets/networkpolicy/gadget.go

```diff
@@ -130,10 +128,8 @@ func (f *Trace) Start(trace *gadgetv1alpha1.Trace) {
 	}
 	f.started = true
 
-	trace.Status.OperationError = ""
 	trace.Status.Output = ""
```

Comment:
Why not at the beginning of the function as you did for the seccomp? Moving it at the beginning would ensure that the `Output` is cleaned up also when the user "restarts" the gadget, or what did I miss?

Reply:
You're right. In this case it's better to move to the beginning.